src/Elasticsearch.Net/ConnectionPool/StaticConnectionPool.cs

```diff
@@ -15,27 +15,35 @@ public StaticConnectionPool(IEnumerable<Uri> uris, bool randomize = true, IDateT
 			: this(uris.Select(uri => new Node(uri)), randomize, dateTimeProvider) { }
 
 		public StaticConnectionPool(IEnumerable<Node> nodes, bool randomize = true, IDateTimeProvider dateTimeProvider = null)
+			: this(nodes, randomize, randomizeSeed: null, dateTimeProvider) { }
+
+		protected StaticConnectionPool(IEnumerable<Node> nodes, bool randomize, int? randomizeSeed = null, IDateTimeProvider dateTimeProvider = null)
```

Comment:
I wonder if for master, we should consider accepting an instance of `Random` here instead of `randomize` and `randomizeSeed`